src/_pytest/runner.py

```diff
@@ -220,13 +224,21 @@ def call_runtest_hook(item, when: "Literal['setup', 'call', 'teardown']", **kwds
 
 @attr.s(repr=False)
 class CallInfo:
-    """ Result/Exception info a function invocation. """
+    """ Result/Exception info a function invocation.
+
+    :param excinfo: (`ExceptionInfo`, optional) describing the captured exception if any.
```

Comment:
The actual format is:

```
:param ExceptionInfo excinfo: describing the captured exception if any.
:param float start: The system time when the call started, in seconds since the epoch.
```

Feel free to follow https://github.com/pytest-dev/pytest/pull/6939/files#r399488974 though and put that in each field if that works. I don't have a strong preference, so it is up to you. 😁

Reply:
thanks @nicoddemus . I did that.